src/Sulu/Bundle/ContactBundle/Controller/AccountMediaController.php

```diff
@@ -49,6 +49,20 @@ public function postAction($id, Request $request)
     }
 
     /**
+     * Replaces the medias of the account.
+     *
+     * @param $id
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function putAction($id, Request $request)
```

Comment:
I think `PUT` should actually not be used on end points without an ID (http://restcookbook.com/HTTP%20Methods/put-vs-post/).

I think the only way to do this restful with one request is to send the data to `/api/accounts/1` as a `PUT` request with all the contact data. We are e.g. also handling the content like this (if you save the excerpt tab of a page also the entire page is sent).